x/gov/types/vote.go

```diff
@@ -13,7 +13,7 @@ import (
 // NewVote creates a new Vote instance
 //nolint:interfacer
 func NewVote(proposalID uint64, voter sdk.AccAddress, options WeightedVoteOptions) Vote {
-	return Vote{proposalID, voter.String(), options}
+	return Vote{ProposalId: proposalID, Voter: voter.String(), Options: options}
```

Comment:
If `len(options) == 1` and the weight is `1`, let's also set `Option` to have a graceful fallback

Reply:
I don't think that's desired.

A user can call `MsgWeightedVote` explicity with `len(options) == 1` and weight == 1, but in state I think we should only save weighted `Options`.

Reply:
I added graceful fallback only for queries in d1d03fd41e1655709e83cf4bf1b3904c093e10c1